spec/spec_helper.rb

```diff
@@ -90,3 +90,5 @@
   Kernel.srand config.seed
 =end
 end
+
+require 'timecop'
```

Comment:
Probably better to either put this at the top of the file near the other require's, or put it in a file in spec/support.  It's usually a good idea to try and keep spec_helper.rb as untouched as possible so that when upgrading the diff between our version and the new one that the new version of rspec wants to use is minimal (e.g. only the changes that rspec wants to make).